src/main/groovy/com/bugsnag/android/gradle/BugsnagProguardConfigTask.groovy

```diff
@@ -17,6 +17,7 @@ class BugsnagProguardConfigTask extends DefaultTask {
     static final String PROGUARD_CONFIG_SETTINGS = """\
     -keepattributes LineNumberTable,SourceFile
     -keep class com.bugsnag.android.NativeInterface { *; }
+    -keep class com.bugsnag.android.Breadcrumb { *; }
     -keep class com.bugsnag.android.Breadcrumbs { *; }
     -keep class com.bugsnag.android.Breadcrumbs\$Breadcrumb { *; }
```

Comment:
I couldnt see `com.bugsnag.android.Breadcrumbs\$Breadcrumb` but maybe i'm missing something?

Reply:
Looks like it was removed in https://github.com/bugsnag/bugsnag-android/commit/45b454465e9ef512d2fa93a3641c6d70aaba1212#diff-326654b6e94ba89f06579d82d3119049

Reply:
Yep. We need to keep it in the ProGuard rules for backwards compatibility with older notifiers.